src/libotutil/ot-keyfile-utils.c

```diff
@@ -60,6 +60,51 @@ ot_keyfile_get_boolean_with_default (GKeyFile *keyfile, const char *section, con
   return TRUE;
 }
 
+// Keep this in sync with
+// https://gitlab.gnome.org/GNOME/glib/-/blob/4a73fbda8be6f80f14b05983eb575c1eb1329c2c/glib/gkeyfile.c?page=5#L4585
+// Except for some reason at some point we added "yes" or "no" as possible values too...
+gboolean
+_ostree_parse_boolean (const char *s, gboolean *out_val, GError **error)
+{
+  g_assert (s);
+  g_assert (out_val);
+
+  if (g_str_equal (s, "yes") || g_str_equal (s, "true") || g_str_equal (s, "1"))
+    {
+      *out_val = TRUE;
+      return TRUE;
+    }
+  else if (g_str_equal (s, "no") || g_str_equal (s, "false") || g_str_equal (s, "0"))
+    {
+      *out_val = FALSE;
+      return TRUE;
+    }
+  return glnx_throw (error, "Invalid boolean: %s", s);
+}
+
+gboolean
+_ostree_parse_tristate (const char *s, OtTristate *out_tri, GError **error)
+{
+  if (strcmp (s, "maybe") == 0)
```

Comment:
This isn't new, but `"maybe"` is a very confusing value to support. Feels like `"auto"` would capture it better?